### PR TITLE
Opensaml update to 2.6.6

### PR DESCRIPTION
**Pull request by GDRDABARERA**

### CODE REVIEW

File: opensaml/2.6.6.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~ KIND, either express or implied.  See the License for the
+~ specific language governing permissions and limitations
+~ under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
```
Comment (thusithathilina) [inline opensaml/2.6.6.wso2v1/pom.xml]: Remove unnecessary empty line

File: opensaml/2.6.6.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+                    <groupId>org.beanshell</groupId>
+                    <artifactId>bsh-core</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.owasp.antisamy</groupId>
+                    <artifactId>antisamy</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
```
Comment (thusithathilina) [inline opensaml/2.6.6.wso2v1/pom.xml]: Remove unnecessary empty line

File: opensaml/2.6.6.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+        <repository>
+            <id>wso2.public.releases</id>
+            <name>WSO2 Public Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public</url>
+        </repository>
+
```
Comment (thusithathilina) [inline opensaml/2.6.6.wso2v1/pom.xml]: Remove unnecessary empty line

**Comment by GDRDABARERA** [comment]
Closed by this commit https://github.com/wso2/orbit/pull/262

**Pull request closed without merging**